mongodb-statefulset-replication.yaml

```diff
@@ -191,7 +191,6 @@ objects:
               # Uncomment this if using dynamic volume provisioning.
               # https://docs.openshift.org/latest/install_config/persistent_storage/dynamically_provisioning_pvs.html
               # volume.alpha.kubernetes.io/storage-class: anything
-              volume.alpha.kubernetes.io/storage-class: dynamic
```

Comment:
@sabre1041 why are we removing this? i don't see anything in the template or documentation that creates the PV for me.